mindmeld/components/nlp.py

```diff
@@ -1636,60 +1638,92 @@ def __init__(
             self.resource_loader, domain, intent, entity_type
         )
         self.entity_resolver = EntityResolverFactory.create_resolver(
-            app_path,
-            entity_type,
-            resource_loader=self.resource_loader
+            app_path, entity_type, resource_loader=self.resource_loader
         )
 
         self.progress_bar = progress_bar
         if isinstance(self.progress_bar, tqdm):
             self.progress_bar.total += 1
 
+    @property
+    def ready(self):
+        # either one can trigger a call to dump method in build_resursive()
+        return self._ready_rc or self._ready_er
```

Comment:
what is _ready_rc? I think both _ready_rc and _ready_er should be renamed more clearly